MrAaronOlsen/trail-up
# Google map features

#### What does  this PR do?
Adds google maps functionality to trail search

#### Where should the reviewer start?
The find trails link

#### How should this be manually tested?
Go to trails link, markers should pop up based on denver center. Use location bar to find new markers in any location. Click on markers for more info.

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran? 
Yes
  - Do Environment Variables need to be set? 
No
  - Any other deploy steps? 
No
